tests/anchor/programs/basic/Cargo.toml

```diff
@@ -13,6 +13,9 @@ name = "basic"
 no-entrypoint = []
 cpi = ["no-entrypoint"]
 
+[patch.crates-io]
+solana_rbpf = { git = "https://github.com/solana-labs/rbpf", tag = "v0.8.0" }
```

Comment:
why is this needed?

Reply:
`solana_rbpf = 0.8.0` is needed for solana 1.17.34 but that version was yanked